src/components/router/FRouterLink.vue

```diff
@@ -24,16 +24,8 @@ template(v-else)
     slot
 </template>
 
-<style lang="stylus"></style>
-
 <script setup lang="ts">
-import { computed } from 'vue';
-import { RouterLink } from 'vue-router';
-
-import FLink from '@/components/FLink.vue';
-import { useSmartLink } from '@/composables/useSmartLink';
-
-import type { RouteLocationRaw } from 'vue-router';
+import type { RouterLink, RouteLocationRaw } from 'vue-router';
```

Comment:
RouterLink isn't a type or am I missing something 😅 

```suggestion
import type { RouteLocationRaw } from 'vue-router';
import { RouterLink } from 'vue-router';
```